src/aks-preview/HISTORY.rst

```diff
@@ -11,7 +11,11 @@ To release a new version, please select a new version number (usually plus 1 to
 
 Pending
 ++++++
+
+0.5.145
```

Comment:
This version number has been used/release, please choose a new one.

Reply:
changed to 0.5.146